examples/graph/plot_DAG_layouts.py

```diff
@@ -0,0 +1,65 @@
+"""
+======================
+DAG in Various Layouts
+======================
+Creates a Directed Acyclic Graph and shows the Graph in various layouts. 
+This can give an idea as to which layout can be chosen for these types of datasets.
+"""
+
+import networkx as nx
+import matplotlib.pyplot as plt
+
+
+G = nx.DiGraph()
+G.add_edges_from(
+    [
+        ("f", "a"),
+        ("a", "b"),
+        ("a", "e"),
+        ("b", "c"),
+        ("b", "d"),
+        ("d", "e"),
+        ("f", "c"),
+        ("f", "g"),
+        ("h", "f"),
+    ]
+)
+
+
+fig, axes = plt.subplots(3, 3, figsize=(25, 12))
+layouts = [
+    None,
+    nx.circular_layout(G),
+    nx.kamada_kawai_layout(G),
+    nx.planar_layout(G),
+    nx.random_layout(G),
+    nx.shell_layout(G),
+    nx.spring_layout(G),
+    nx.spectral_layout(G),
+    nx.spiral_layout(G),
+]
+titles = [
+    "No Layout",
```

Comment:
Calling this "No layout" is misleading as `draw_networkx` actually uses the `spring_layout` by default when `pos=None`.

Reply:
I failed to notice the default value Sir. Have made the edits. Thanks for pointing it out👍